tests/endtoend/test_sql_functions.py

```diff
@@ -1,10 +1,12 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 import json
+from unittest import skip
 
-from azure_functions_worker import testutils
+from tests.utils import testutils
 
 
+@skip("Skipping till library version containing SQL is released.")
```

Comment:
No need to implement it - but can we think of a skip where if the package contains SQL, this should still run? As the beta versions have the SQL implementation.